maps/defficiency.dmm

```diff
@@ -8568,7 +8568,7 @@
 "diN" = (/turf/simulated/shuttle/wall{icon_state = "wall3"; dir = 1},/area/shuttle/vox/station)
 "diO" = (/obj/machinery/computer/security,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
 "diP" = (/obj/machinery/computer/shuttle_control/vox,/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
-"diQ" = (/obj/machinery/computer/crew{holomap_filter = HOLOMAP_FILTER_VOX},/turf/simulated/shuttle/floor/vox{icon_state = "floor4"},/area/shuttle/vox/station)
```

Comment:
🤔

Reply:
🤷‍♂

Reply:
I guess @PaulRitter wrote this manually and DM doesn't like it, should be fine as long as HOLOMAP_FILTER_VOX is actually 16